hphp/runtime/vm/jit/vasm-simplify.cpp

```diff
@@ -431,6 +431,21 @@ bool simplify(Env& env, const copyargs& inst, Vlabel b, size_t i) {
   });
 }
 
+bool simplify(Env& env, const loadb& inst, Vlabel b, size_t i) {
+  return if_inst<Vinstr::movzbl> (env, b, i+1, [&] (const movzbl& cm) {
```

Comment:
Two nits:
1. No space after function names before the paren.
2. Space between operators (`i + 1`).